packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -255,6 +255,9 @@ fi
 %endif
 
 %changelog
+* Wed Jan 15 2020 Yunhee Lee <yhlee@amazon.com> - 1.36.1-1
+- Cache Agent version 1.36.1
+- Fix potential file descriptor leak from context logger
```

Comment:
is this needed in init? since the changes are in agent

Reply:
It's in the release instructions, and looks like previous releases have updated also.

Reply:
I'll remove agent fix details.